OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+- deads2k
```

Comment:
please remove deads2k and lavalamp as they're not involved, please add me instead as second owner since although I haven't done ruby, I am involved with kubernetes-client.